java/ql/lib/semmle/code/java/frameworks/spring/SpringSecurity.qll

```diff
@@ -0,0 +1,124 @@
+/**
+ * Provides classes for working with Spring classes and interfaces from
+ * `org.springframework.security.*`.
+ */
+
+import java
+
+/** The class `org.springframework.security.config.annotation.web.builders.HttpSecurity`. */
+class TypeHttpSecurity extends Class {
```

Comment:
Same as above. Would it make sense to consider changing the public class names to be more closely associated with Spring?